var/spack/repos/builtin/packages/ima-evm-utils/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ImaEvmUtils(AutotoolsPackage):
+    """IMA/EVM control utilities."""
+
+    homepage = "http://linux-ima.sourceforge.net"
+    url      = "http://sourceforge.net/projects/linux-ima/files/ima-evm-utils/ima-evm-utils-1.2.1.tar.gz"
```

Comment:
Please use `https` for the url and, ideally, for the homepage when possible.

```suggestion
    url      = "https://sourceforge.net/projects/linux-ima/files/ima-evm-utils/ima-evm-utils-1.2.1.tar.gz"
```